app/commands/unlock_command.rb

```diff
@@ -9,8 +9,26 @@ def run
       return Slash.reply(ValidationErrorMessage.build(record: env)) if env.invalid?
 
       slashdeploy.unlock_environment(user.user, env)
-      Slash.say UnlockedMessage.build \
-        environment: env
+      new_active_lock = slashdeploy.give_lock_to_next_user(env)
```

Comment:
This should happen as part of `SlashDeploy::Service#unlock_environment` (we'd still want the lock to transition to the next user in other control layers).